### DISCUSSION TITLE
how to change the intro sound into the TCD?

### DISCUSSION THREAD

**Discussion by Gabin37** [CircuitSetup/Time-Circuits-Display#47] [Q&A]
Hi everyone and happy new year to all of you :)
Is there a way to change the intro sound ( clock sound) with for exemple, the "alarmon" sound instead when powering on?
Thanks for your help :)

Best regards

**Comment by realA10001986** [comment]
https://github.com/realA10001986/Time-Circuits-Display#sound-file-substitution

**Comment by Gabin37** [comment]
Thanks so much! I did not see this :)
